oauthenticator/gitlab.py

```diff
@@ -23,10 +24,30 @@
 
 from .oauth2 import OAuthLoginHandler, OAuthenticator
 
+GITLAB_URL = os.getenv('GITLAB_URL')
+GITLAB_HOST = os.getenv('GITLAB_HOST')
+
+if not GITLAB_URL and GITLAB_HOST:
+    warnings.warn('Use of GITLAB_HOST has been deprecated in v0.8. '
+                  'Rename GITLAB_HOST environemnt variable to GITLAB_URL.',
+                  DeprecationWarning)
+    if GITLAB_HOST.startswith('https://') or \
+            GITLAB_HOST.startswith('http://'):
+        GITLAB_URL = GITLAB_HOST
+    else:
+        # Hides common mistage of users which set the GITLAB_HOST
+        # without a protocol specification.
+        GITLAB_URL = 'https://{0}'.format(GITLAB_HOST)
```

Comment:
The warning is always displayed. This only helps new developers that don't read docs carefully and they will get the warning anyway.

Reply:
This line will generate a duplication of https:// if GITLAB_HOST is set correctly.